deploy/kube-config/influxdb/grafana-service.yaml

```diff
@@ -2,16 +2,17 @@ apiVersion: v1
 kind: Service
 metadata:
   labels:
-    kubernetes.io/cluster-service: 'true'
```

Comment:
Why remove the cluster label from the Grafana service? I know that it might cause issue for a service that isn't part of a Kubelet manifest but it could be indicated to the user.